recipes/c-ares/all/conanfile.py

```diff
@@ -0,0 +1,60 @@
+# coding=utf-8
```

Comment:
```suggestion
```
https://www.python.org/dev/peps/pep-0263/ UTF-8 is default on Python3

Reply:
This recipe can also be consumed by Conan running py2. I think the encoding could be useful